repository.py

```diff
@@ -324,10 +326,12 @@ def _repo_config(self):
                 outfh = open(key_path, "w")
                 outfh.write(infh.read())
                 return """
-gpgcheck=1
-repo_gpgcheck=1
+gpgcheck=%s
+repo_gpgcheck=%s
 gpgkey=file://%s
-""" % (key_path)
+""" % (1 if self.__gpg_check else 0,
+       1 if self.__repo_gpg_check else 0,
```

Comment:
Can be simplified using `int(self.__gpg_check)`.

Reply:
right